### PR TITLE
Adding offline part for elytron tls commands

### PR DESCRIPTION
**Pull request by OndrejKotek**

### CODE REVIEW

File: commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/tls/AddLdapKeyStore.java [hunk 1]
```diff
@@ -111,6 +117,25 @@ public void apply(OnlineCommandContext ctx) throws Exception {
...
+                .subtree("elytronSubsystem", Subtree.subsystem("elytron"))
+                .parameter("atrName", name)
+                .parameter("atrDirContext", dirContext)
+                .parameter("atrSearchPath", searchPath)
+                .parameter("atrSearchRecursive", searchRecursive)
+                .parameter("atrSearchTimeLimit", searchTimeLimit)
+                .parameter("atrFilterAlias", filterAlias)
+                .parameter("atrFilterCertificate", filterCertificate)
+                .parameter("atrFilterIterate", filterIterate)
+                .parameter("atrAliasAttribute", null)
```
Comment (jtymel) [inline commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/tls/AddLdapKeyStore.java]: is it intentionally here? It looks odd to me at the first sight

File: testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddLdapKeyStoreOfflineTest.java [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                .dirContext("dirContext")
+                .searchPath("searchPath")
+                .build();
+
+        assertXmlIdentical(SUBSYSTEM_SIMPLE, Files.toString(cfg, Charsets.UTF_8));
+        client.apply(addLdapKeyStore);
+        assertXmlIdentical(SUBSYSTEM_SECOND_KEY_STORE, Files.toString(cfg, Charsets.UTF_8));
+    }
+
+//    @Test
```
Comment (jtymel) [inline testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddLdapKeyStoreOfflineTest.java]: Is this test ingored intentionally? If so, could you please write a brief reason for that?

File: testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddServerSSLContextOfflineTest.java [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+
+public class AddServerSSLContextOfflineTest {
```
Comment (jtymel) [inline testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddServerSSLContextOfflineTest.java:18]: Key manager used in a couple of places (constants, methods, fail descriptions)

File: testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddTrustManagerOfflineTest.java [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+
+public class AddTrustManagerOfflineTest {
```
Comment (jtymel) [inline testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddTrustManagerOfflineTest.java:18]: Key stores and key managers used on a couple of places (constants, method names, fail descriptions), I would prefer usage of trust stores and trust managers

File: testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddClientSSLContextOfflineTest.java [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+
+public class AddClientSSLContextOfflineTest {
```
Comment (jtymel) [inline testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddClientSSLContextOfflineTest.java:18]: Key managers and key stores used on a couple of places (constants, method names, fail descriptions)

**Review by jtymel — COMMENTED** [review]
I've spotted only a couple of minor issues. Thank you, really great work!

**Comment by OndrejKotek** [comment]
Great catches, thx! Going to fix and merge.

**Pull request merged**